extension-ui/background.js

```diff
@@ -82,3 +82,23 @@ chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
     return true; // כדי להורות שהתגובה היא אסינכרונית
   }
 });
+
+function showNotification(title, message) {
+  chrome.notifications.create({
+      type: 'basic',
```

Comment:
please change to notification that contains the text:
Notice: 
browsing time in {site} is about to reach limitation in {num} minutes.

the texts should be saved in constants file. site & num are parameters.